src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml

```diff
@@ -249,7 +249,7 @@
                                     <controls:DropDownButton x:Name="PART_DatePickerIconDropDownButton"
                                                              DockPanel.Dock="Right"
                                                              IsArrowVisible="False"
-                                                             TabIndex="30"
+                                                             KeyboardNavigation.TabIndex="30"
```

Comment:
yes...this is a magic value... previously it was 30...i think it's better leave as it was